app/templates/events/event_list.html

```diff
@@ -69,22 +70,62 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
     </thead>
     <tbody>
         {% for event in events %}
+        {% set Fade_event= "" %}
```

Comment:
We use camelCase everywhere for things like this so for consistency change this to fadeEvent.